docs/docs/mdx/programmatically-creating-pages.md

```diff
@@ -197,15 +197,15 @@ exports.createPages = async ({ graphql, actions, reporter }) => {
   // Create blog post pages.
   const posts = result.data.allMdx.edges
 
-  // We'll call `createPage` for each result
+  // you'll call `createPage` for each result
```

Comment:
you => You _(Capitalize)_

Reply:
These changes are copied over from the original English. @nucliweb If they're not fixed already, you could submit a PR against [that repo](https://github.com/gatsbyjs/gatsby) and they'll be applied at the next sync.